test/zepto.html

```diff
@@ -2664,11 +2670,16 @@
             })
           })
         })
-        t.assertTrue(fired)
-        t.assertIdentical(Zepto, arg1)
-        t.assertIdentical(Zepto, arg2)
-        t.assertIdentical(Zepto, arg3)
-        t.assertIdentical(Zepto, arg4)
+        t.pause()
+        setTimeout(function(){
+          t.resume(function(){
+            t.assertTrue(fired)
+            t.assertIdentical(Zepto, arg1)
+            t.assertIdentical(Zepto, arg2)
+            t.assertIdentical(Zepto, arg3)
+            t.assertIdentical(Zepto, arg4)
+          })
+        }, 500)
```

Comment:
Even one (non-nested) invocation would be tricky in the same way. We need to add check to the task queue in the way that it will executed after ready callback. NodeJS `process.nextTick` would helps us, but we don’t have such feature in pure JS. There are two possible approaches to reduce test suite execution: remove the test case or try to find some minimal stable value.

* Remove the test case. It more or less worth as (after the commit) we don’t provide guarantee that a callback will executed syncronously and immediatelly. That’s what the test case was about.
* Minimal stable value. 50ms seems unstable for me (FF 47.0 on Linux, CPU P8400), but 100ms seems stable.

I don’t know which approach is better. Do you have suggestions on this?